File: Python/8py 2/nth_even.py
"""
|--------------------------------------------------------------------------
| Get Nth Even Number
|--------------------------------------------------------------------------
|
"""

def nth_even(n):
   return n * 2 - 2


nth_even = lambda _:_*2-2


def nth_even(n):
   return (n + n) - 2 


def nth_even(n):
   return 2 * (n - 1)


print(nth_even(1), 0)
print(nth_even(2), 2)
print(nth_even(3), 4)
print(nth_even(100), 198)
print(nth_even(1298734), 2597466)

File: PHP/8php 1/nthEven.php
<?php
/*
|--------------------------------------------------------------------------
| Get Nth Even Number
|--------------------------------------------------------------------------
|
*/

function nthEven($n) {
   return $n * 2 - 2;
}


function nthEven($n) {
   return --$n << 1;
}


function nthEven($n) {
   return ($n - 1) * 2;
}


print_r(nthEven(1) . ' ' . 0 . '
');
print_r(nthEven(2) . ' ' . 2 . '
');
print_r(nthEven(3) . ' ' . 4 . '
');